docs/ci-integration/guides/codebuild-integration.md

```diff
@@ -21,7 +21,7 @@ phases:
       - wget https://github.com/earthly/earthly/releases/download/v0.6.28/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly
   pre_build:
     commands:
-      - echo Logging in to Docker
+      - echo Logging into Docker
```

Comment:
This correction is my newest and has been the most controversial of late. Happy to drop.